### PR TITLE
CV Eval - Deletion

### PR DESCRIPTION
**Pull request by sambible**
Big PR here, lots of removals. This is deleting all cases marked as such as part of the CV Component Eval.

If you'd like to, you can review the cases deleted in the Pheonix Component Feedback sheet. The primary reason for deletion for a case is either
1) Dupe coverage that is picked up in another case
2) Older Test that isn't longer relevant ( i.e. Pulp upgrade made some cases irrelevant )

### CODE REVIEW

**Comment by Griffin-Sullivan** [comment]
> Older Test that isn't longer relevant ( i.e. Pulp upgrade made some cases irrelevant )

I tried bringing this problem up in a call once and I don't think I got the point across. I imagine this is an issue we will need to find a good solution for whether it's automated or a new process for the team each time we branch.

File: tests/foreman/api/test_contentviewfilter.py [hunk 1]
```diff
@@ -65,43 +62,6 @@ def content_view_module_stream(module_org, sync_repo_module_stream, module_targe
...
-        """
-        response = client.get(
-            target_sat.api.AbstractContentViewFilter().path(),
-            auth=get_credentials(),
-            verify=False,
-        )
-        assert response.status_code == http.client.OK
-
-    @pytest.mark.tier2
-    def test_negative_get_with_bad_args(self, target_sat):
```
Comment (vijaysawant) [inline tests/foreman/api/test_contentviewfilter.py:87]: Is this scenario already been covered in any other test? Any reason to remove?

File: tests/foreman/api/test_contentview.py [hunk 2]
```diff
@@ -942,85 +822,6 @@ def initiate_testclass(
...
-
-        :expectedresults: RH Content assigned and present in a view
-
-        :CaseImportance: High
-        """
-        assert len(self.yumcv.repository) == 1
-        assert self.yumcv.repository[0].read().name == REPOS['rhst7']['name']
-
-    @pytest.mark.tier2
-    def test_positive_add_rh_custom_spin(self, target_sat):
```
Comment (vijaysawant) [inline tests/foreman/api/test_contentview.py:959]: Is this test consider by mistake, I don't see any recommendation in sheet.

**Review by vijaysawant — COMMENTED** [review]
Looked over 2 files only
`api/test_contentview.py` & `test_contentviewfilter.py`
query for @sambible

File: tests/foreman/api/test_contentviewfilter.py (hunk 1 above)
Comment (sambible) [reply tests/foreman/api/test_contentviewfilter.py:87]: I believe this was removed as it was considered a low value test case

File: tests/foreman/cli/test_contentviewfilter.py [hunk 3]
```diff
@@ -788,78 +788,6 @@ def test_negative_update_with_name(self, new_name, content_view, module_target_s
                 {'content-view-id': content_view['id'], 'name': new_name}
             )
 
-    @pytest.mark.tier1
-    def test_negative_update_with_same_name(self, module_org, content_view, module_target_sat):
```
Comment (vijaysawant) [inline tests/foreman/cli/test_contentviewfilter.py:792]: I believe this deletion must be due to low severity of test, don't see any qe-recommendation in the sheet.

File: tests/foreman/cli/test_contentviewfilter.py [hunk 4]
```diff
@@ -788,78 +788,6 @@ def test_negative_update_with_name(self, new_name, content_view, module_target_s
...
-            module_target_sat.cli.ContentView.filter.update(
-                {
-                    'content-view-id': content_view['id'],
-                    'name': new_name,
-                    'new-name': cvf_name,
-                }
-            )
-
-    @pytest.mark.tier1
-    def test_negative_update_inclusion(self, module_org, content_view, module_target_sat):
```
Comment (vijaysawant) [inline tests/foreman/cli/test_contentviewfilter.py:830]: I believe this deletion must be due to low severity of test, don't see any qe-recommendation in the sheet.

**Review by vijaysawant — APPROVED** [review]
Ack.
I cross verified all deleted tests with tests present in sheet.
Put my understanding for 2 tests as part review, please take a look.

File: tests/foreman/cli/test_docker.py [hunk 5]
```diff
@@ -589,182 +462,6 @@ def test_positive_publish_with_docker_repo_composite(
...
-        assert len(content_view['versions']) == 0
-
-        publish_amount = randint(2, 5)
-        for _ in range(publish_amount):
-            module_target_sat.cli.ContentView.publish({'id': content_view['id']})
-        content_view = module_target_sat.cli.ContentView.info({'id': content_view['id']})
-        assert len(content_view['versions']) == publish_amount
-
-    @pytest.mark.tier2
-    def test_positive_publish_multiple_with_docker_repo_composite(
```
[the remark this answers is not in the document]
Comment (sambible) [reply tests/foreman/cli/test_docker.py:612]: Yea, I'm removing the whole file in another PR

File: tests/foreman/cli/test_contentviewfilter.py (hunk 3 above)
Comment (sambible) [reply tests/foreman/cli/test_contentviewfilter.py:793]: marked as QE delete

File: tests/foreman/cli/test_contentviewfilter.py (hunk 4 above)
Comment (sambible) [reply tests/foreman/cli/test_contentviewfilter.py:831]: also listed as delete on sheet

File: tests/foreman/api/test_contentview.py (hunk 2 above)
Comment (sambible) [reply tests/foreman/api/test_contentview.py:1067]: just added it back.

**Review by vsedmik — APPROVED** [review]

**Pull request merged**